data/audit_log.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kubermatic.k8s.io/v1
```

Comment:
Isn't this entire file redundant, given that `data/seed.yaml` exists (and is kept up-to-date, while the `audit_log.yaml` will be out-of-sync soon)?

Reply:
seeds.yaml is very large, this is a simple snippet to show the basic structure.

Reply:
But it will still get out-of-sync quickly. This example file is also not something a user can just copy-paste and apply straight-away to their cluster, so it's not a minimal viable example.

Reply:
Ok, will remove it.